### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by musiciancodes**
ran black on all files to clean things up

### CODE REVIEW

**Comment by musiciancodes** [comment]
added poetry & formatted files

File: backend/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+name = "twofish"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+django = "^3.0.5"
+
+[tool.poetry.dev-dependencies]
```
Comment (Lucretiel) [inline backend/pyproject.toml:12]: This would be the place to add `black`

**Review by Lucretiel — CHANGES_REQUESTED** [review]
Recommend reverting the changes in /covid/migrations, since those files are auto-generated. In the PR that adds black we should configure it to ignore that directory permanently.

File: backend/pyproject.toml (hunk 1 above)
Comment (musiciancodes) [reply backend/pyproject.toml:12]: done.

**Comment by musiciancodes** [comment]
also reverted migrations formatting.

**Comment by musiciancodes** [comment]
@Lucretiel ping on review?

**Review by Lucretiel — APPROVED** [review]

**Pull request merged**